pytest/lib/bridge.py

```diff
@@ -137,11 +137,11 @@ def __init__(self,
         self.custom = dict()
 
 def atexit_cleanup(obj):
-    print("Cleaning %s on script exit" % (obj.__class__.__name__))
+    logger.info("Cleaning %s on script exit" % (obj.__class__.__name__))
```

Comment:
logging does %-formatting so this would be better written as:

    logger.info('Cleaning %s on script exit', type(obj).__name__)

Here and throughout.

Reply:
I think it'd be consistent to just change these all over to just f-strings and not have all the gotchas I mentioned, but was trying not to get this ticket out of scope by doing that. Since this ticket was to just change over to logging